bleach/__init__.py

```diff
@@ -18,9 +16,9 @@
 
 
 # yyyymmdd
-__releasedate__ = "20200918"
+__releasedate__ = ""
 # x.y.z or x.y.z.dev0 -- semver
-__version__ = "3.2.1"
+__version__ = "4.0.0.dev0"
```

Comment:
should we release a beta first?

Reply:
I think the nicety of an alpha/beta/rc comes only when you've got people lined up to use/test it that are representative of Bleach users. If you don't, then you spend a bunch of time doing the alpha/beta/rc, wait a bunch of time, and then it's not clear whether that extra effort bought you anything or reduced the chances of a follow-up fixit release.

In the past, I've kept tabs on bugs and done follow-up fixit releases. I can usually do that pretty quickly.

If it were me, I'd release on a Monday with a few following days that I could turn around a fixit release and not go the alpha/beta/rc route.